shared/src/commonMain/kotlin/com/vml/tutorial/plantshop/profilePreferences/presentation/UserInput.kt

```diff
@@ -0,0 +1,49 @@
+package com.vml.tutorial.plantshop.profilePreferences.presentation
+
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.foundation.text.KeyboardActions
+import androidx.compose.foundation.text.KeyboardOptions
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.material3.TextField
+import androidx.compose.material3.TextFieldDefaults
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.focus.FocusDirection
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.LocalFocusManager
+import androidx.compose.ui.text.input.VisualTransformation
+import androidx.compose.ui.unit.dp
+
+@Composable
+fun UserInput(
+    state: String?,
```

Comment:
do you mean value?

Reply:
Yes, I probably forgot to rename it. Done :)